lighthouse-core/audits/metrics/cumulative-layout-shift.js

```diff
@@ -46,9 +46,9 @@ class CumulativeLayoutShift extends Audit {
     return {
       // Calibrated to assure 0.1 gets a score of 0.9. https://web.dev/cls/#what-is-a-good-cls-score
```

Comment:
we should really consider reworking our math so we can define in terms of the 90 cutoff :) haha